features/steps/user-management.rb

```diff
@@ -0,0 +1,281 @@
+  step 'I do not see a Delete button' do
+    expect(page).to have_no_button("Delete")
+  end
+
+  step 'I am sent to the Change User Information page' do
+    expect(page.title).to eq("Modify Pirate Profile")
+  end
+
+  step 'I am sent to the Profile page' do
+    expect(page.title).to eq("Motley User")
+    expect(page.text).to match(Regexp.new(".*#{@user.name}.*"))
+  end
+
+  step 'I am sent to the Sign In page' do
+    expect(page.title).to eq("Sign In")
+  end
+
+  step 'I see a success message containing "Signed in successfully"' do
+    has_flash_msg(severity: :notice, containing: "Signed in successfully")
+  end
+
+  step 'I see an alert containing "You must be signed in to access that page"' do
+    has_flash_msg(severity: :alert, containing: "You must be signed in to access that page")
+  end
+
+  step 'I see an alert containing "You must be an admin user to access that page"' do
+    has_flash_msg(severity: :alert, containing: "You must be an admin user to access that page")
+  end
+
+  step 'I see an alert containing "Invalid email or password"' do
+    has_flash_msg(severity: :alert, containing: "Invalid email or password")
+  end
+
+  step 'I see the new user name' do
+    expect(page).to have_content(@user.name)
+  end
+
+  step 'I click Edit' do
+    # need test to see that the page is for the right user
+    click_on "Edit Info"
+    sync_page
+  end
+
+  step 'I click Edit for that other user' do
+    find(".user-id-#{@another_user.id} form.button_to .edit").click
+    sync_page
+  end
+
+  step 'I click Delete and confirm deletion for that other user' do
+    my_accept_alert do
+      find(".user-id-#{@another_user.id} form.button_to .delete").click
+    end
+    sync_page
+  end
+
+  step 'that other user is deleted' do
+    user = User.find_by(email: @another_user.email)
+    expect(user).to be_nil
+  end
+
+  step 'there is at least one other user' do
+    @another_user = FactoryGirl.create(:user)
+  end
+
+  step 'I see information for another user' do
+    expect(page.text).to have_content(@another_user.name)
+  end
+
+  step 'I fill in the fields' do
+    @start_date = "9-Jul-2010"
+    @another_user = FactoryGirl.build(:user)
+    fill_in "user_name", with: @another_user.name
+    fill_in "user_tone_name", with: @another_user.tone_name
+    select_date(@start_date)
+    fill_in "user_email", with: @another_user.email
+    fill_in "user_password", with: PASSWORD
+    fill_in "user_password_confirmation", with: PASSWORD
+    check "user_admin"
+  end
+
+  step 'I click Sign Up' do
+    click_link_or_button "Sign up"
+    sync_page
+  end
+
+  step 'the account is created' do
+    user = User.find_by(email: @another_user.email)
+    expect(user).not_to be_nil
+  end
+
+  step 'I change the mutable fields' do
+    change_mutable_fields(@user)
+  end
+
+  step 'I change the mutable fields for that other user' do
+    change_mutable_fields(@another_user)
+  end
+
+  step 'I change the admin checkbox' do
+    if @user.admin?
```

Comment:
I have explicit tests for both the admin and non-admin cases that use this to ensure I am not colliding.